cmd/tkey-verification/main.go

```diff
@@ -81,35 +57,25 @@ Known firmwares:
 		"Read verification data from a file located in `DIRECTORY` and named after the TKey UDI in hex, instead of from a URL. You can for example first use \"verify --show-url\" and download the verification file manually on some other computer, then transfer the file back and use \"verify --base-dir .\" (command: verify).")
 	pflag.StringVar(&baseURL, "base-url", defaultBaseURL,
 		"Set the base `URL` of verification server for fetching verification data (command: verify).")
+	pflag.StringVarP(&binPath, "app", "a", "",
+		"`PATH` to the device app to show vendor signing pubkey (command: show-pubkey).")
 	pflag.BoolVar(&versionOnly, "version", false, "Output version information.")
+	pflag.BoolVar(&build, "build", false, "Output build data about included device apps and firmwares")
 	pflag.BoolVar(&helpOnly, "help", false, "Output this help.")
-	pflag.Usage = func() {
-		desc := fmt.Sprintf(`Usage: %s command [flags...]
-
-Commands:
-  serve-signer  Run the server that offers an API for creating vendor signatures.
-
-  remote-sign   Call the remote signing server to sign for a local TKey.
-
-  verify        Verify that a TKey is genuine by extracting the TKey UDI and using it
-                to fetch the verification data, including tag and signature from the
-                web. Then running the correct verisigner-app on the TKey, extracting
-                the public key and verifying it using the vendor's signing public key.
-
-                The flags --show-url and --base-dir can be used to show the URL for
-                downloading the verification data on one machine, and verifying the
-                TKey on another machine that lacks network, see more below.`, progname)
-
-		le.Printf("%s\n\nFlags:\n%s\n%s", desc, pflag.CommandLine.FlagUsagesWrapped(86), builtWith)
-	}
+	pflag.Usage = usage
```

Comment:
There is no `--speed` flag. Right now this prevents tkey-verification to be runnable on macos, since QEMU for some reason (a bug I guess) uses baudrate of 9600 baud.